scripts/travis_upload_nightlies.sh

```diff
@@ -2,6 +2,15 @@
 
 set -e
 
+# Don't upload anything unless it's a push commit to the master branch.
```

Comment:
can be nice to test unmerged branches, or staging branches especially.

Reply:
ah but you always have them available on github build artifacts too. pick any run here: https://github.com/digitalbitbox/bitbox-wallet-app/actions

and github even builds and upload in forks. like mine here: https://github.com/x1ddos/bitbox-wallet-app/actions/runs/153204239

this file, travis_upload_nightlies.sh is run on travis-ci only. or did you mean something else?

Reply:
True! All good.